### PR TITLE
Add new topic `aliens` for GraphQL issues

### LINKED ISSUE #1301: [2pt] Show an issue if there are alien uuids alive in membership
It means there are two different clusters with the same cookie. And it's no good.

### PR DESCRIPTION
**Pull request by HustonMmmavr**
I didn't forget about

- [x] Tests
- [x] Changelog
- [x] Documentation

Close #1301

### CODE REVIEW

File: cartridge/issues.lua [hunk 1]
```diff
@@ -427,6 +427,25 @@ local function list_on_cluster()
         end
     end
 
+    -- Check aliens in membership
+
+    for uri, member in membership.pairs() do
+        local uuid = member.payload.uuid
+        if member.status ~= 'left' and uuid ~= nil
+            and topology_cfg.servers[uuid] == nil
+        then
```
Comment (rosik) [inline cartridge/issues.lua:436]:
```suggestion
        local uuid = member.payload.uuid
        if member.status == 'alive'
        and uuid ~= nil
        and topology_cfg.servers[uuid]
        then
```

File: cartridge/issues.lua [hunk 2]
```diff
@@ -427,6 +427,25 @@ local function list_on_cluster()
...
+            table.insert(ret, {
+                level = 'warning',
+                topic = 'aliens',
+                message = string.format(
+                    'Unknown instance at membership table %s',
```
Comment (rosik) [inline cartridge/issues.lua:441]:
```suggestion
                    'Instance %s with alien uuid is in the membership',
```

File: cartridge/issues.lua (hunk 1 above)
Comment (rosik) [reply cartridge/issues.lua:436]: We're not interested in dead members

File: test/integration/issues_test.lua [hunk 3]
```diff
@@ -253,3 +253,37 @@ function g.test_state_hangs()
...
+    -- reset replica2 membership data
+    replica2.net_box:eval([[
+        local uuid = ...
+        local membership = require('membership')
+        _G.__old_member_data = membership.myself().payload
+        membership.init('localhost', 13309)
+        membership.set_payload('uuid', uuid)
+        membership.set_payload('alias', 'alien')
+        membership.probe_uri('localhost:13301')
+    ]], {helpers.uuid('b', 'b', 1)})
```
Comment (rosik) [inline test/integration/issues_test.lua:270]: set_payload(uuid) + probe_uri is enough

File: test/integration/issues_test.lua [hunk 4]
```diff
@@ -253,3 +253,37 @@ function g.test_state_hangs()
...
+
+    t.assert_equals(helpers.list_cluster_issues(master), {{
+        level = 'warning',
+        topic = 'aliens',
+        message = 'Unknown instance at membership table localhost:13309 (alien)',
+    }})
+
+    -- restore replica2 membership data
+    replica2.net_box:eval([[
```
Comment (rosik) [inline test/integration/issues_test.lua:279]: I'd suggest killing it and waiting untill the alien issue disappears. And then restart it.

**Review by rosik — COMMENTED** [review]
Please, update the doc on the top of the `issues.lua` file too.

File: cartridge/issues.lua (hunk 1 above)
Comment (HustonMmmavr) [reply cartridge/issues.lua]: Done

File: cartridge/issues.lua (hunk 2 above)
Comment (HustonMmmavr) [reply cartridge/issues.lua]: Fixed

File: test/integration/issues_test.lua (hunk 3 above)
Comment (HustonMmmavr) [reply test/integration/issues_test.lua]: Test was refactored

File: test/integration/issues_test.lua (hunk 4 above)
Comment (HustonMmmavr) [reply test/integration/issues_test.lua]: Test was refactored

**Pull request merged**